openssh.yaml

```diff
@@ -100,6 +102,17 @@ subpackages:
           mkdir -p "${{targets.subpkgdir}}"/usr/lib/ssh
           mv "${{targets.destdir}}"/usr/lib/ssh/sftp-server "${{targets.subpkgdir}}"/usr/lib/ssh/
 
+  - name: "openssh-pam"
+    description: "OpenSSH PAM"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/lib/ssh
+          mkdir -p "${{targets.subpkgdir}}"/etc/pam.d
+          mkdir -p "${{targets.subpkgdir}}"/usr/sbin
```

Comment:
The sshd.pam config file shouldn't need to go into /usr/sbin. It can be installed into /etc/pam.d/ directly.

However the example one does not work fully.